### PR TITLE
Document QC checks for EMC

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/8168

### CODE REVIEW

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:2]: Add "at the source" or "by the external providers"

"is not included in Mondo until the content is updated **at the source/by the data providers** and passes the QC checks."

Comment (sabrinatoro) [reply docs/editors-guide/externally-managed-content.md:2]: Maybe add something after the first sentence to explain that we do that so we don't block the update. 
Suggestion: 
**To limit delay due to QC failure,** any content from external providers (...)

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:7]: The description is confusing. 
is this reporting a mondo term?

Suggestion: 
The synonym from the external source is the same as a synonym already reported on a Mondo term, but the synonym scope is not the same. For example, the synonym from the source is "exact", and the synonym in Mondo is "broad".

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:8]: Suggestion: 
A database_cross_reference in the EMC file is not in CURIE format.

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:9]: Suggestion: 
An exact synonym, excluding abbreviations, or label is used in Mondo for two different Mondo terms/entities.

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:10]: how is this one different from duplicate_scoped_synonym ?

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:11]: I am confused by this one.  Do we get files from external sources that do not have any x-ref and have only synonyms? Is this what this is checking? 

if so, 
A synonym from the external source file does not have a database cross reference (...)

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:12]: Is this a EMC related QC check? I am assuming that we have a check to make sure that we remove the extra whitespace when we process these files. 

Do we really exclude information and tell the source to remove trailing whitespace? (that seems a bit weird to me)

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:13]: Again, is this really a QC for the EMC?

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:14]: "to the same **external source** term'

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel I am confused by this document. 
- If this document is for the external sources, then it needs more explanation about what external managed content is, how it is process, etc. But I can set this on the side for now and we can update later. 
- Generally, I think the descriptions need to be clearer about what information is from the external source, and what information is in Mondo. For example "a synonym from the external source is already in Mondo but with a different scope". Therefore the synonym is ignored until it can be reviewed (in Mondo and in the external source)
- I am a bit confused about the QC check contents. I thought these would be the one reported [here](https://docs.google.com/document/d/1SLCIIxbW-B-cXfg15A6Xpq9oQ9t0yaHJ_yVkaQia__c/edit#heading=h.fgkun4my6zai). However, there is a lot of other QC checks in this document which seem more general and not specific to the EMC. For example: the white space trailing.  I guess my question is: are these truly specific to the externally managed content? (if the answer is yes, then it is all good.)

**Comment by twhetzel** [comment]
I don't know why the QC checks for EMC are not the ones in listed in the [doc](https://docs.google.com/document/d/1SLCIIxbW-B-cXfg15A6Xpq9oQ9t0yaHJ_yVkaQia__c/edit#heading=h.fgkun4my6zai) you referenced. The Mondo QC checks that are done for EMC are based on the failures of EMC content that has occurred during the development of this pipeline. These are all the checks of things I've had to fix manually when Nico would submit a PR for adding this content.

**Comment by twhetzel** [comment]
Also, I do not know if it is still the plan to email the report to providers or not.

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:2]: updated

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:7]: updated

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:8]: updated

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:9]: updated: An exact synonym, excluding abbreviations, or label in the EMC file duplicates an existing exact synonym or label for a different Mondo term/entity.

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:10]: good call, since duplicate_scoped_synonym includes this check (qc-related-exact-synonym) I removed it here

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:14]: updated

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:13]: i think this did happen once and is why it's listed here

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:12]: well, this is an existing QC check for Mondo so if the trailing whitespace is not fixed at the source each month a curator will need to fix this manually. this is added because it has happened in past additions of EMC content

Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:12]: however, i also agree that removing any leading/trailing whitespace should be something that happens when processing the EMC content

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:11]: let me look at this one again....

File: docs/editors-guide/externally-managed-content.md
Comment (sabrinatoro) [inline docs/editors-guide/externally-managed-content.md:8]: EMC is not defined in the document yet. We should have it in the first paragraph so we know that EMC is Externally Managed Content.

**Review by sabrinatoro — APPROVED** [review]
@twhetzel My only comment is that EMC needs to be defined early in the text so we know what it stands for. Otherwise, this iteration is fine. 

I am approving so you can merge after having added what EMC stands for, without waiting for another review.

File: docs/editors-guide/externally-managed-content.md
Comment (twhetzel) [reply docs/editors-guide/externally-managed-content.md:8]: done

**Pull request merged**
